js/src/forum/components/TokensManager.js

```diff
@@ -0,0 +1,62 @@
+import Component from '../../common/Component';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import Button from '../../common/components/Button';
+import humanTime from '../../common/helpers/humanTime';
+
+export default class TokensManager extends Component {
+  init() {
+    this.refresh();
+  }
+
+  refresh() {
+    this.tokens = null;
+
+    app.request({
+      method: 'GET',
+      url: app.forum.attribute('apiUrl') + '/tokens',
+    }).then(payload => {
+      this.tokens = app.store.pushPayload(payload);
+      m.redraw();
+    });
+
+    // TODO: pagination
+  }
+
+  view() {
+    if (this.tokens === null) {
+      return LoadingIndicator.component();
+    }
+
+    console.log(this.tokens);
+
+    return m('table', [
```

Comment:
Why not JSX?

Reply:
It's all WIP right now, I'll fix the code style later.

I'm used to not using JSX for reasons I have already expressed elsewhere, but I'll adapt to match the Flarum code style.